pkgs/applications/audio/mopidy/subidy.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, pythonPackages, mopidy }:
+
+pythonPackages.buildPythonApplication rec {
+  pname = "mopidy-subidy";
+  version = "1.0.0";
+
+  src = fetchurl {
+    url = "https://github.com/Prior99/${pname}/archive/${version}.tar.gz";
+    sha256 = "b9a33af87751a19efd3defcbe20383ff4f8c900996b3eadee07ca26739c3a7fc";
+  };
+
+  propagatedBuildInputs = [ mopidy pythonPackages.py-sonic];
+
+  doCheck = false;
```

Comment:
Why are checks disabled? Does upstream not contain checks?

Reply:
I've corrected this